### PR TITLE
Finalized Medical HUD Interface Update

### PR DESCRIPTION
**Pull request by Puckaboo2**
It's the pull request #6110 but I removed flashing skulls and edited the comments in the code. I'm a dumb fluff; I forgot git amend existed and shut the last PR down.

### CODE REVIEW

**Comment by Puckaboo2** [comment]
I need to get my shit together. Damn.

File: code/modules/client/preference/loadout/loadout_hat.dm [hunk 1]
```diff
@@ -117,3 +117,13 @@
 	display_name = "science beret"
 	path = /obj/item/clothing/head/beret/sci
 	allowed_roles = list("Research Director", "Scientist")
+
+/datum/gear/hat/surgicalcap_purple
```
Comment (Fox-McCloud) [inline code/modules/client/preference/loadout/loadout_hat.dm]: These aren't acceptable to add in this PR.

**Comment by Puckaboo2** [comment]
I don't know how to separate them from this PR. I was hoping they would separate themselves, but they didn't. Can you help me?

**Comment by Fox-McCloud** [comment]
You should always create a new branch when working on a new project to avoid situations like this--which varies depending on which client you're using. I'm guessing you're using Github's stock client, so here's some documentation on it: https://help.github.com/desktop/guides/contributing/creating-a-branch-for-your-work/

You'll want your master to be hard reset before doing any of this...and yes, that does mean you will lose your progress, but you reallyyyy shouldn't be making changes to master, generally speaking.

```
git fetch upstream
git reset --hard upstream/master
git push -f
``` 
in the command line of your github client is one way to you can hard reset, unless @Crazylemon64 has another method/approach he'd like to suggest.

File: code/game/data_huds.dm [hunk 2]
```diff
@@ -68,47 +68,55 @@
...
+			return "health-60"
+		if(-80 to -70)
+			return "health-70" //Doc?
+		if(-90 to -80)
+			return "health-80" //Hey, doc?
+		if(-100 to -90)
+			return "health-90" //HURRY UP, DOC!
 		else
-			return "health-100"
+			return "health-100" //doc u had 1 job
```
Comment (Crazylemon64) [inline code/game/data_huds.dm]: :smirk:

**Pull request merged**